packages/types/src/clerk.ts

```diff
@@ -751,6 +774,13 @@ export interface Clerk {
    * initiated outside of the Clerk class.
    */
   __internal_setActiveInProgress: boolean;
+
+  /**
+   * API Keys Object
+   * @experimental
+   * This API is in early access and may change in future releases.
+   */
+  apiKeys: APIKeysNamespace;
```

Comment:
I know I called this out before. In this case we usually do 
```suggestion
  __experimental_apiKeys: APIKeysNamespace;
```

but feel free to ignore, if this is communicated on the docs.